shared/constants/metametrics.js

```diff
@@ -178,6 +178,8 @@
  * @property {'three_box_enabled'} THREE_BOX_ENABLED - when 3box feature is
  *  toggled we identify the 3box_enabled trait
  * @property {'theme'} THEME - when the user's theme changes we identify the theme trait
+ * @property {'token_detection_enabled'} [TOKEN_DETECTION_ENABLED] - when token detection feature is toggled we
+ * identify the token_detection_enabled trait
```

Comment:
There is another jsdoc type that needs to be updated in this file that describes the shape of the traits object versus this one that describes the shape of the TRAITS constant object